docs/src/examples/components/Menu/States/MenuExampleDisabled.shorthand.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { Menu } from '@stardust-ui/react'
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Disabled Item', disabled: true },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+const MenuExampleDisabled = () => <Menu defaultActiveIndex={0} items={items} />
+
+export default MenuExampleDisabled
```

Comment:
```suggestion
export default MenuExampleItemDisabled
```

May be this name will be more obvious?

Reply:
Removing this example, it has been already added by @codepretty in another PR.